src/Sylius/Bundle/CoreBundle/Tests/Responses/Expected/authentication/access_denied.json

```diff
@@ -0,0 +1,4 @@
+{
+    "error": "access_denied",
+    "error_description": "OAuth2 authentication required"
```

Comment:
Kinda sucks that we have different format for most errors and 403 but we can take care of that later.